elastic-tube-3d/metadata.yaml

```
@@ -0,0 +1,36 @@
+name: Elastic tube 3D
+path: elastic-tube-3d # relative to git repo 
+url: https://precice.org/tutorials-elastic-tube-3d.html
+clean: ./clean-tutorial.sh 
+tags: # just exemplary here 
+  - nearest-projection
+  - IMVJ
+  - FSI
```

A better example here would be:

```suggestion
tags: # just exemplary here 
  - long
  - nearest-projection
  - IMVJ
```

The idea of the tags is that we will be able to run specific tests on demand (e.g., only run the "long" tutorials nightly/weekly), or even on specific runners (e.g., for cases requiring licensed codes).

We could also use these tags for specific features (e.g., `nearest-projection` or `IMVJ`), but I see this as secondary in priority.

Tags such as `FSI` probably do not help us much, as they would make the corresponding checks quite fine-grained, without a clear use case.